doc/source/tutorial/optimize.rst

```diff
@@ -2068,44 +2068,59 @@ Either of the two approaches outlined above can be used, although it's strongly
 advised to supply the map-like callable due to the overhead of creating new processes.
 Performance gains will only be made if the objective function is expensive to
 calculate.
+Let's compare how much parallelization can help compared to the serial version. To
+simulate a slow function we use the ``time`` package.
 
 ::
 
-    >>> x0 = np.array([2.0, 3.0, 4.0, 5.0])
-    >>> with Pool(2) as pwl:
-    ...     res = minimize(rosen, x0, method='L-BFGS-B', options={'workers':pwl.map})
-    >>> res.x
-    array([0.99999903, 0.99999808, 0.99999614, 0.99999228])  # may vary
+    >>> import time
+    >>> def slow_func(x):
+    ...     time.sleep(0.0002)
+    ...     return rosen(x)
+
+Examine the serial minimization first::
+
+    In [1]: rng = np.random.default_rng()
```

Comment:
Nit, it is slightly weird how the prompt style changes from `>>>` to the ipython style halfway through the example

Reply:
I know. I looked at the integrate example. I think it's the use of %timeit that causes problems when one uses >>>